apps/eth_geth_adapter/lib/eth_geth_adapter/worker.ex

```diff
@@ -124,7 +123,10 @@ defmodule EthGethAdapter.Worker do
     {:reply, ErrorHandler.errors(), reg}
   end
 
-  def handle_call({:boot_adapter, datadir}, _from, reg) do
-    {:reply, GethManager.start(datadir), reg}
+  # `EthGethAdapter.GethManager` is only compiled and used in tests.
+  if Application.get_env(:ewallet, :env) == :test do
```

Comment:
```suggestion
  if Application.get_env(:eth_geth_adapter, :env) == :test do
```

Then in `apps/eth_geth_adapter/config/config.exs`:

```elixir
config :eth_geth_adapter,
  env: Mix.env()
```